docker-compose.yml

```
@@ -37,7 +39,9 @@ services:
       - VNC_NO_PASSWORD=1
     ports:
       - '4444:4444'
-      - '5900:5900'
+      - "5900:5900"
```

```suggestion
      - '5900:5900'
```
single comma looks consistent with previous port declaration